src/FSharpLint.Core/Rules/Conventions/UsedUnderscorePrefixedElements.fs

```diff
@@ -0,0 +1,56 @@
+module FSharpLint.Rules.UsedUnderscorePrefixedElements
+
+open System
+open FSharpLint.Framework
+open FSharpLint.Framework.Suggestion
+open FSharp.Compiler.Syntax
+open FSharpLint.Framework.Ast
+open FSharpLint.Framework.Rules
+
+let private checkUsedIdent (previousIdent: Ident) (body: SynExpr) =
+    match body with
+    | SynExpr.Sequential(debugPointAtSequential, isTrueSeq, synExpr, expr2, range) ->
+        match synExpr with
+        | SynExpr.App(exprAtomicFlag, isInfix, funcExpr, argExpr, range) ->
+            match argExpr with
+            | SynExpr.Ident ident ->
+                if previousIdent.idText = ident.idText then
+                    {
+                        Range = range
+                        Message = String.Format(Resources.GetString ("RulesUsedUnderscorePrefixedElements"))
+                        SuggestedFix = None
+                        TypeChecks = []
```

Comment:
@realmarv please use List.Empty instead of `[]`

Reply:
Done